spec/manageiq/cloud_watch_spec.rb

```diff
@@ -25,7 +25,7 @@
       expect(CloudWatchLogger::Client::AWS_SDK::DeliveryThreadManager).to receive(:new).and_return(double("CloudWatchLogger::Client::AWS_SDK::DeliveryThreadManager", :deliver => nil))
     end
 
-    it "returns a CloudWatch::Client" do
+    xit "returns a CloudWatch::Client" do
```

Comment:
Did you want to un-xit these before merge?

Reply:
Right, is this and the following test supposed to pass or fail?  I understand the interface used to return a new logger but now can return self.  Not sure if the intention is to rewrite this test or something else.

Reply:
Ah right forgot about this guy, so I don't know that it is important that this actually return a CloudWatch::Client and now it'll return a `ActiveSupport::BroadcastLogger` with 7.1.  The only way to support 6.1,7.0 and 7.1 is to delete this test and have the interface for CloudWatch logger not always return a CloudWatch::Client

Reply:
NOTE It is a little odd that this automatically builds and wraps a ContainerLogger, that seems like something you'd want the caller to do?  Maybe we change this to simply build a `CloudWatchLogger::Client` and let the caller wrap with whatever they want?  E.g. if we used this on an appliance we'd want to wrap it with a journal logger not a container logger.

Reply:
I opened https://github.com/ManageIQ/manageiq-loggers/pull/79 to simplify the CloudWatch logger and let the caller chose any broadcast logging they want.  For this I think I'm going to just delete these tests so we can keep the existing behavior and still support 6.1 .. 7.1

Reply:
Note, we already pass log_group and log_stream for cloud watch, I can see passing the type of logging as an option: journal or container.

Reply:
Yeah that is an option however that is still a little limiting.
What if you didn't want to broadcast anywhere else? (Could pass a nil)
What if you wanted to broadcast to a non-manageiq-loggers logger? (Could pass a logger instance)
At that point isn't that just a wrapper around the existing `.wrap` method?